signalbot/bot.py

```diff
@@ -533,7 +538,7 @@ async def _consume_new_item(self, name: int) -> None:
             context = Context(self, message)
             await command.handle(context)
         except Exception as e:
-            logging.error(f"[{command.__class__.__name__}] {e.__class__.__name__}: {e}")
+            traceback.print_exc()
```

Comment:
Does this print the same errors as before? The previous error was useful because it was associated with a command name.

Reply:
Take a look at what I switched to. I really appreciate getting a backtrace as well as the exception message, so I format one but send it through the logger tagged with the command name to keep the spirit of what you have currently.